libp2p/Host.cpp

```diff
@@ -91,15 +92,17 @@ Host::Host(string const& _clientVersion, KeyPair const& _alias, NetworkConfig co
                      // simultaneously
     m_tcp4Acceptor(m_ioService),
     m_runTimer(m_ioService),
-    m_alias(_alias),
+    m_alias(_secretAndENR.first),
+    m_enr(_secretAndENR.second),
     m_lastPing(chrono::steady_clock::time_point::min()),
     m_capabilityHost(createCapabilityHost(*this))
 {
     cnetnote << "Id: " << id();
+    cnetnote << "ENR: " << m_enr;
```

Comment:
In the interest of consistency should we call enr() here? (since we call id() rather than m_alias.pub())